pages/faq/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import faqContent from "@/lib/faq/faqContent"
+import FaqSection from "@/components/FaqSection"
+
+const Faq = () => (
+  <div className="mt-20 text-white w-4/5 mx-auto">
+    <div className="space-y-6">
+      <h1 className="font-semibold text-2xl">Frequently Asked Questions (FAQs)</h1>
+
+      {/* Map through the termsContent array */}
```

Comment:
remove any dev comments
- rule: code should be self-documenting

learn more: https://swimm.io/learn/documentation-tools/tips-for-creating-self-documenting-code